lib/receptor_controller/client/configuration.rb

```diff
@@ -42,7 +42,7 @@ def initialize
       @queue_auto_ack    = false
       @queue_host        = nil
       @queue_max_bytes   = nil
-      @queue_persist_ref = nil
+      @queue_persist_ref = "receptor-client.#{SecureRandom.uuid}"
```

Comment:
According to the document the end result of making persist_ref unique per pod is the same as `nil`. The code path in ruby kafka client is different for each case though. I will be very surprised to see adding persist_ref helps solving the reported problem.

Reply:
@bzwei Do you mean that when the consumer group is created it doesn't have access to older messages?

Reply:
@bzwei With the persist_ref (group_id) it allows us to debug the consumer groups and the offsets that are being used. We are seeing that we are missing messages even with ack set to True and persist_ref is nil. By adding a persist_ref we would be able to see the offsets for each of the consumer group.

Reply:
@mkanoor added